docs/users.md

```diff
@@ -0,0 +1,78 @@
+# Users
+
+This is a collection of user methods. 
+
+## ToC
+
+- [`sdk.users.me()`](#me)
+- [`sdk.users.get(id)`](#getById)
+- [`sdk.users.lookup(email)`](#lookByEmail)
+
+<a id="me"></a>
+
+## `sdk.users.me()`
+This method is used to get details about the current logged in user.
+
+### API
+```js
+sdk.users.me(): Promise<User>
+```
+
+### Example
+
+```js
+const eventbrite = require('eventbrite');
+
+// Create configured Eventbrite SDK
+const sdk = eventbrite({token: 'OATH_TOKEN_HERE'});
+
+sdk.users.me().then((user) => {
+    console.log(`Hi ${user.name}!`);
+});
+```
+
+<a id="getById"></a>
+
+## `sdk.users.get(id)`
+This method is used to load the details for a specific user by their user id. 
+
+### API
+```js
+sdk.users.get(id: number|string): Promise<User>
+```
+
+### Example
+
+```js
+const eventbrite = require('eventbrite');
+
+// Create configured Eventbrite SDK
+const sdk = eventbrite({token: 'OATH_TOKEN_HERE'});
+
+sdk.users.get(1234567890).then((user) => {
+    console.log(`Hi ${user.name}!`);
+});
+```
+
+<a id="lookByEmail"></a>
+
+## `sdk.users.lookup(email)`
```

Comment:
I'm thinking we should call this `emailLookup` just in case there end up being other ways to look up a user? Also, I'm pretty sure this is an undocumented feature. I'd still include it though because it's public, but you won't find any docs for it.

Reply:
should we rename this to be specific or have a generic lookup, that we adapt over time?

Reply:
If we have a generic lookup then it'll have to take an object with an `email` property so we can have it adapt w/o changing the function signature

Reply:
Right, is that more or less preferred?